layouts/partials/tree.html

```diff
@@ -1,25 +1,27 @@
-{{ $section := . }}
```

Comment:
@bep Can you take a quick look and make sure this looks good to you? Thx

Reply:
This does not look good.